app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+
+import { Formik, Form } from "formik";
+import { Button, Pane, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import { CONTACTS_FORM_VALIDATION_SCHEMA, ROLES } from "../constants";
+
+const NoteForm = ({ onClose, contact }) => {
+  const [submitted, setSubmitted] = useState(false);
+
+  const handleSubmit = () => {
+    setSubmitted(true);
+    Toastr.success("Contact added successfully.");
+    onClose();
+  };
+
+  return (
+    <Formik
+      initialValues={contact}
+      validateOnBlur={submitted}
+      validateOnChange={submitted}
+      validationSchema={CONTACTS_FORM_VALIDATION_SCHEMA}
+      onSubmit={handleSubmit}
+    >
+      {({ isSubmitting }) => (
+        <Form className="w-full">
+          <Pane.Body className="space-y-6">
+            <div className="flex w-full justify-between align-middle">
+              <Input
+                className="mr-5 w-full"
+                label="First Name"
+                name="firstName"
+                placeholder="Enter First Name"
+              />
+              <Input
+                className="w-full"
+                label="Last Name"
+                name="lastName"
+                placeholder="Enter Last Name"
+              />
+            </div>
+            <Input
+              className="w-full flex-grow-0"
+              label="Email Address"
+              name="email"
+              placeholder="Enter your email address"
+            />
+            <Select
+              isSearchable
+              className="w-full flex-grow-0"
+              label="Role"
+              name="role"
+              options={ROLES}
+              placeholder="Select Role"
+            />
+          </Pane.Body>
+          <Pane.Footer>
+            <Button
+              className="mr-3"
+              disabled={isSubmitting}
+              label="Save Changes"
+              loading={isSubmitting}
+              size="large"
+              style="primary"
+              type="submit"
+              onClick={() => setSubmitted(true)}
+            />
+            <Button
+              label="Cancel"
+              size="large"
+              style="text"
+              onClick={onClose}
+            />
```

Comment:
Go through the Slack thread once again and fix the issue with this piece of code.

Reply:
Fixed